python/pyspark/sql/functions.py

```diff
@@ -1116,12 +1160,12 @@ def from_utc_timestamp(timestamp, tz):
 @since(1.5)
 def to_utc_timestamp(timestamp, tz):
     """
-    Given a timestamp, which corresponds to a certain time of day in the given timezone, returns
```

Comment:
Let's revert this change or do this the same thing to Scala/R too if it matters.